plugins/poll/assets/javascripts/controllers/poll-ui-builder.js.es6

```diff
@@ -38,6 +42,11 @@ export default Controller.extend({
     ];
   },
 
+  @computed("chartType", "pollType", "numberPollType")
+  isPie(chartType, pollType, numberPollType) {
+    return pollType !== numberPollType && chartType === "pie";
```

Comment:
We've been moving away from string constants because we find them error prone. It's okay for the variable to be a string but could you compare it against a constant of some sort?

Reply:
Are you saying to do  this?
```
return pollType !== "number" && chartType === "pie";
```

Reply:
No I think he thinks of something like this:

```
    const PIE_TYPE_IDENTIFIER = "pie";

    // code omitted

    return pollType !== numberPollType && chartType === PIE_TYPE_IDENTIFIER;
```

Reply:
Yes what @jjaffeux said!